miqa/core/models/scan_note.py

```diff
@@ -15,6 +15,7 @@ class Meta:
     # We want to set created to arbitrary values during import
     created = models.DateTimeField(default=timezone.now)
     modified = ModificationDateTimeField()
-    creator = models.ForeignKey(User, on_delete=models.PROTECT)
+    creator = models.ForeignKey(User, on_delete=models.PROTECT, null=True, blank=True)
+    initials = models.CharField(max_length=2)
```

Comment:
At first glance I thought this was just a denormalization that could be replaced with

```python
@property
def initials(self) -> str:
    if self.creator:
        return f'{self.creator.first_name[0]}{self.creator.last_name[0]}'
    return '??'
```

But now I see that the initials are able to be imported, so this model makes sense. Carry on.